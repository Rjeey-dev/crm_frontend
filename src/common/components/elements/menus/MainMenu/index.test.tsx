import { shallow } from 'enzyme';
import React from 'react';

import HeaderLinks from './index';

it("Renders <HeaderLinks/> with correct properties", () => {
    const output = shallow(
        <HeaderLinks/>
    ).html();

    expect(output).toMatchSnapshot();
});
