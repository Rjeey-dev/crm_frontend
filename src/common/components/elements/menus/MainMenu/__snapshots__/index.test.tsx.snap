// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <HeaderLinks/> with correct properties 1`] = `"<ul class=\\"navbar-nav navbar-nav-left header-links d-none d-md-flex\\"><li class=\\"nav-item\\"><a href=\\"#\\" class=\\"nav-link\\"><span class=\\"badge badge-primary ml-1\\"></span>Missing translationId: menus.header_links.shedule for language: \${ languageCode }</a></li><li class=\\"nav-item\\"><a href=\\"#\\" class=\\"nav-link\\"><i class=\\"mdi mdi-elevation-rise\\"></i>Missing translationId: menus.header_links.reports for language: \${ languageCode }</a></li><li class=\\"nav-item\\"><a href=\\"#\\" class=\\"nav-link\\"><i class=\\"mdi mdi-bookmark-plus-outline\\"></i>Missing translationId: menus.header_links.score for language: \${ languageCode }</a></li></ul>"`;
